packages/nice_parser/nice_parser.1.0.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
I had a look at the code and currently, this line doesn't do anything. You need to define your examples as tests in your dune file

Reply:
Are you sure? On my machine (dune version 1.11.0) it *does* run the tests. Note the `(inline_tests)` attribute in the `example/dune` file.

Reply:
oh you're right, my bad.